src/Traversal/Sdk/Sdk.targets

```diff
@@ -114,11 +114,23 @@
           DependsOnTargets="$(BuildDependsOn)"
           Condition=" '$(IsGraphBuild)' != 'true' "
           Returns="@(CollectedBuildOutput)">
+
+    <ItemGroup>
```

Comment:
Is this necessary, can't you just call `AnyHaveMetadataValue` on @(ProjectReference) on line 125?

Reply:
We could, but then we wouldn't be honoring `BuildInParallel` fallback property. Because we could do `AnyHaveMetadataValue('BuildInParallel', 'true')` but then I realized that would be a more complicated implementation because if a project doesn't have metadata value then we need to honor `BuildInParallel`. 

Then for the other cases where we have more properties like `CleanInParallel`... it gets more complicated, so I figured this was the simplest, to basically do the same as we do when dispatching to the MSBuild task.